core/tests/test_apps.bitcoin.authorization.py

```diff
@@ -0,0 +1,109 @@
+from common import unittest, H_
+
+from trezor.messages.AuthorizeCoinJoin import AuthorizeCoinJoin
+from trezor.messages.GetOwnershipProof import GetOwnershipProof
+from trezor.messages.SignTx import SignTx
+from trezor.messages import InputScriptType
+
+from apps.common import coins
+from apps.bitcoin.authorization import CoinJoinAuthorization
+
+_ROUND_ID_LEN = 32
+
+
+class TestAuthorization(unittest.TestCase):
+
+    coin = coins.by_name('Bitcoin')
+
+    msg_auth = AuthorizeCoinJoin(
+        coordinator="www.example.com",
+        max_total_fee=40000,
+        fee_per_anonymity=0.003 * 10**9,
+        address_n=[H_(84), H_(0), H_(0)],
+        coin_name=coin.coin_name,
+        script_type=InputScriptType.SPENDWITNESS,
+    )
+
+    authorization = CoinJoinAuthorization(msg_auth, None, coin)
```

Comment:
please move `msg_auth` and `authorization` to `def setUp(self)` -- so that we know that each testcase has a fresh instance

Reply:
https://github.com/trezor/trezor-firmware/pull/1127/commits/e9831759a8fc2912e47757487f7befb89f5d272e